app/controllers/games_controller.rb

```diff
@@ -16,15 +17,11 @@ def new
 
   def create
     @game = Game.new(game_params)
-
     @game.white_player_id = current_user.id
     @game.black_player_id = current_user.id
-    @game.game_status = "Unknown"
 
     if @game.save
-      puts "Save Successful"
-      render :show, status: :created
-      #redirect_to @game, notice: 'Game was successfully created.'
+      redirect_to @game, status: :created, notice: 'Game was successfully created.'
```

Comment:
Correct. This will need to be fixed later. Or if you know the quick solution, we can fix it now.